### PR TITLE
Making shortcode templates more readable by fixing spaces in parentheses

### PR DESCRIPTION
**Pull request by whs-dot-hk**

### CODE REVIEW

File: content/en/templates/shortcode-templates.md [hunk 1]
```diff
@@ -88,14 +88,14 @@ For the second position, you would just use:
 `with` is great when the output depends on a parameter being set:
 
 ```
-{{ with .Get "class"}} class="{{.}}"{{ end }}
+{{ with .Get "class" }} class="{{.}}"{{ end }}
```
Comment (jmooring) [inline content/en/templates/shortcode-templates.md:91]:
```suggestion
{{ with .Get "class" }} class="{{ . }}"{{ end }}
```

File: content/en/templates/shortcode-templates.md [hunk 2]
```diff
@@ -88,14 +88,14 @@ For the second position, you would just use:
...
 ```
 
 `.Get` can also be used to check if a parameter has been provided. This is
 most helpful when the condition depends on either of the values, or both:
 
 ```
-{{ if or (.Get "title") (.Get "alt") }} alt="{{ with .Get "alt"}}{{.}}{{else}}{{.Get "title"}}{{end}}"{{ end }}
+{{ if or (.Get "title") (.Get "alt") }} alt="{{ with .Get "alt" }}{{.}}{{ else }}{{ .Get "title" }}{{ end }}"{{ end }}
```
Comment (jmooring) [inline content/en/templates/shortcode-templates.md:98]:
```suggestion
{{ if or (.Get "title") (.Get "alt") }} alt="{{ with .Get "alt" }}{{ . }}{{ else }}{{ .Get "title" }}{{ end }}"{{ end }}
```

File: content/en/templates/shortcode-templates.md [hunk 3]
```diff
@@ -211,17 +211,17 @@ You have created the shortcode at `/layouts/shortcodes/img.html`, which loads th
 {{< code file="/layouts/shortcodes/img.html" >}}
 <!-- image -->
 <figure {{ with .Get "class" }}class="{{.}}"{{ end }}>
-    {{ with .Get "link"}}<a href="{{.}}">{{ end }}
-        <img src="{{ .Get "src" }}" {{ if or (.Get "alt") (.Get "caption") }}alt="{{ with .Get "alt"}}{{.}}{{else}}{{ .Get "caption" }}{{ end }}"{{ end }} />
-    {{ if .Get "link"}}</a>{{ end }}
-    {{ if or (or (.Get "title") (.Get "caption")) (.Get "attr")}}
+    {{ with .Get "link" }}<a href="{{.}}">{{ end }}
```
Comment (jmooring) [inline content/en/templates/shortcode-templates.md:214]:
```suggestion
    {{ with .Get "link" }}<a href="{{ . }}">{{ end }}
```

File: content/en/templates/shortcode-templates.md [hunk 4]
```diff
@@ -211,17 +211,17 @@ You have created the shortcode at `/layouts/shortcodes/img.html`, which loads th
...
+        <img src="{{ .Get "src" }}" {{ if or (.Get "alt") (.Get "caption") }}alt="{{ with .Get "alt" }}{{.}}{{ else }}{{ .Get "caption" }}{{ end }}"{{ end }} />
```
Comment (jmooring) [inline content/en/templates/shortcode-templates.md:215]:
```suggestion
        <img src="{{ .Get "src" }}" {{ if or (.Get "alt") (.Get "caption") }}alt="{{ with .Get "alt" }}{{ . }}{{ else }}{{ .Get "caption" }}{{ end }}"{{ end }} />
```

File: content/en/templates/shortcode-templates.md [hunk 5]
```diff
@@ -211,17 +211,17 @@ You have created the shortcode at `/layouts/shortcodes/img.html`, which loads th
...
+    {{ if .Get "link" }}</a>{{ end }}
+    {{ if or (or (.Get "title") (.Get "caption")) (.Get "attr") }}
     <figcaption>{{ if isset .Params "title" }}
         <h4>{{ .Get "title" }}</h4>{{ end }}
-        {{ if or (.Get "caption") (.Get "attr")}}<p>
+        {{ if or (.Get "caption") (.Get "attr") }}<p>
         {{ .Get "caption" }}
-        {{ with .Get "attrlink"}}<a href="{{.}}"> {{ end }}
+        {{ with .Get "attrlink" }}<a href="{{.}}"> {{ end }}
```
Comment (jmooring) [inline content/en/templates/shortcode-templates.md:222]:
```suggestion
        {{ with .Get "attrlink" }}<a href="{{ . }}"> {{ end }}
```

**Review by jmooring — CHANGES_REQUESTED** [review]

**Comment by whs-dot-hk** [comment]
@jmooring Thanks

**Review by jmooring — APPROVED** [review]

**Pull request merged**